lib/rails_response_dumper/runner.rb

```diff
@@ -103,7 +103,8 @@ def run_dumps
             dump = {
               request: {
                 method: request.method,
-                url: request.url
+                url: request.url,
+                headers: request.headers.to_h
```

Comment:
Perhaps we should call this `env` to make it more clear that these aren't HTTP headers but are Rack env variables.

```suggestion
                env: request.headers.to_h
```

Reply:
See most recent push; these are now more specifically limited to standard CGI headers.

Reply:
These still are not HTTP headers, so I continue to think it'd be more accurate to call this "env" after the Rack environment.

For example, the HTTP header is "Host" or "host" (case insensitive) and not "HTTP_HOST".

Reply:
Once the property is accurately named, any downside to including he entire data structure?

Reply:
> Once the property is accurately named, any downside to including he entire data structure?

Looks like this would pull in non-serializable objects, so no need to include the entire data structure.

Reply:
@jdufresne After further research, it looks like `ActionDispatch::Http::Headers::CGI_VARIABLES` only covers the HTTP header variables *not* prefixed by "HTTP_", so my filter should more accurately be:
```
dump[:request][:headers].delete_if { |key, _| !(key.in?(ActionDispatch::Http::Headers::CGI_VARIABLES) || key =~ /^HTTP_/) }
```
Given everything, what are your thoughts on the key name instead being `header_variables:` (perhaps my preference), or `action_dispatch_headers:` or `rack_headers:` or `rack_header_variables:` ?
I don't want to lose the fact that these are environment variables — as this is honest to their actual source and their formatting — but I also don't want to hide the fact that their filtering and purpose is that of an HTTP header set.